packages/contracts/deploy/deploy.ts

```diff
@@ -242,6 +242,17 @@ export const deployUnirep = async (
     )
     await lazyMerkleContract.deployed()
 
+    const pairingPath = 'contracts/libraries/Pairing.sol/Pairing.json'
+    const pairingArtifacts = tryPath(pairingPath)
+    const _pairingFactory = new ethers.ContractFactory(
+        pairingArtifacts.abi,
+        pairingArtifacts.bytecode,
+        deployer
+    )
+    const pairingFactory = await GlobalFactory(_pairingFactory)
+    const pairingContract = await retryAsNeeded(() => pairingFactory.deploy())
+    await pairingContract.deployed()
```

Comment:
this deployment is not required.
It should be compiled with verifiers.

Reply:
@vivianjeng 
Oh,Sorry.
I deleted it.

Reply:
It's okay!
thank you for your contribution 🙏🏻